From 39752f3f0d1f8a437b7055ab312ea344c6a5deda Mon Sep 17 00:00:00 2001
From: Fabien Dessenne <fabien.dessenne@foss.st.com>
Date: Wed, 6 Apr 2022 09:25:00 +0200
Subject: [PATCH 0046/1141] pinctrl: stm32: update pin function names for
 stm32mp257

Update stm32mp257 pin function names according to v5.47 "pinout synthesis"
spec (17 Mar 22).

Change-Id: I6a0407dd33aaad44a1b6f8cc3af8ef200e952f6f
Signed-off-by: Fabien Dessenne <fabien.dessenne@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/247157
Reviewed-by: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 drivers/pinctrl/stm32/pinctrl-stm32mp257.c | 22 +++++++++++-----------
 1 file changed, 11 insertions(+), 11 deletions(-)

diff --git a/drivers/pinctrl/stm32/pinctrl-stm32mp257.c b/drivers/pinctrl/stm32/pinctrl-stm32mp257.c
index 91dffc12768b..d408ecc31674 100644
--- a/drivers/pinctrl/stm32/pinctrl-stm32mp257.c
+++ b/drivers/pinctrl/stm32/pinctrl-stm32mp257.c
@@ -514,7 +514,7 @@ static const struct stm32_desc_pin stm32mp257_pins[] = {
 	STM32_PIN(
 		PINCTRL_PIN(37, "PC5"),
 		STM32_FUNCTION(0, "GPIOC5"),
-		STM32_FUNCTION(3, "SPDIFRX1_IN2"),
+		STM32_FUNCTION(3, "SPDIFRX1_IN1"),
 		STM32_FUNCTION(6, "MDF1_SDI1"),
 		STM32_FUNCTION(9, "TIM8_CH1N"),
 		STM32_FUNCTION(10, "I2C4_SDA"),
@@ -530,7 +530,7 @@ static const struct stm32_desc_pin stm32mp257_pins[] = {
 		PINCTRL_PIN(38, "PC6"),
 		STM32_FUNCTION(0, "GPIOC6"),
 		STM32_FUNCTION(2, "RTC_REFIN"),
-		STM32_FUNCTION(3, "SPDIFRX1_IN1"),
+		STM32_FUNCTION(3, "SPDIFRX1_IN0"),
 		STM32_FUNCTION(6, "MDF1_CKI1"),
 		STM32_FUNCTION(9, "TIM8_CH1"),
 		STM32_FUNCTION(10, "I2C4_SCL"),
@@ -843,7 +843,7 @@ static const struct stm32_desc_pin stm32mp257_pins[] = {
 		STM32_FUNCTION(0, "GPIOD12"),
 		STM32_FUNCTION(2, "SPI7_MISO"),
 		STM32_FUNCTION(3, "SPI2_MISO I2S2_SDI"),
-		STM32_FUNCTION(4, "SPDIFRX1_IN3"),
+		STM32_FUNCTION(4, "SPDIFRX1_IN2"),
 		STM32_FUNCTION(6, "UART8_RTS"),
 		STM32_FUNCTION(10, "TIM4_ETR"),
 		STM32_FUNCTION(11, "SDMMC3_CMD"),
@@ -975,7 +975,7 @@ static const struct stm32_desc_pin stm32mp257_pins[] = {
 		PINCTRL_PIN(70, "PE6"),
 		STM32_FUNCTION(0, "GPIOE6"),
 		STM32_FUNCTION(2, "SPI4_RDY"),
-		STM32_FUNCTION(5, "SPDIFRX1_IN3"),
+		STM32_FUNCTION(5, "SPDIFRX1_IN2"),
 		STM32_FUNCTION(7, "USART1_TX"),
 		STM32_FUNCTION(9, "TIM1_ETR"),
 		STM32_FUNCTION(12, "FMC_AD1 FMC_D1"),
@@ -988,7 +988,7 @@ static const struct stm32_desc_pin stm32mp257_pins[] = {
 		PINCTRL_PIN(71, "PE7"),
 		STM32_FUNCTION(0, "GPIOE7"),
 		STM32_FUNCTION(4, "SAI4_D4"),
-		STM32_FUNCTION(5, "SPDIFRX1_IN4"),
+		STM32_FUNCTION(5, "SPDIFRX1_IN3"),
 		STM32_FUNCTION(7, "USART1_RX"),
 		STM32_FUNCTION(9, "TIM1_CH4N"),
 		STM32_FUNCTION(11, "TIM14_CH1"),
@@ -1208,7 +1208,7 @@ static const struct stm32_desc_pin stm32mp257_pins[] = {
 	STM32_PIN(
 		PINCTRL_PIN(87, "PF7"),
 		STM32_FUNCTION(0, "GPIOF7"),
-		STM32_FUNCTION(3, "SPDIFRX1_IN2"),
+		STM32_FUNCTION(3, "SPDIFRX1_IN1"),
 		STM32_FUNCTION(4, "SPI6_SCK"),
 		STM32_FUNCTION(5, "SAI3_SD_A"),
 		STM32_FUNCTION(8, "TIM2_ETR"),
@@ -1263,7 +1263,7 @@ static const struct stm32_desc_pin stm32mp257_pins[] = {
 		PINCTRL_PIN(91, "PF11"),
 		STM32_FUNCTION(0, "GPIOF11"),
 		STM32_FUNCTION(2, "MCO1"),
-		STM32_FUNCTION(3, "SPDIFRX1_IN1"),
+		STM32_FUNCTION(3, "SPDIFRX1_IN0"),
 		STM32_FUNCTION(4, "SPI6_RDY"),
 		STM32_FUNCTION(5, "SAI2_SCK_A"),
 		STM32_FUNCTION(6, "MDF1_SDI6"),
@@ -1569,7 +1569,7 @@ static const struct stm32_desc_pin stm32mp257_pins[] = {
 		STM32_FUNCTION(0, "GPIOH2"),
 		STM32_FUNCTION(2, "LPTIM2_CH1"),
 		STM32_FUNCTION(3, "SPI7_RDY"),
-		STM32_FUNCTION(4, "SPDIFRX1_IN4"),
+		STM32_FUNCTION(4, "SPDIFRX1_IN3"),
 		STM32_FUNCTION(5, "SAI1_SCK_B"),
 		STM32_FUNCTION(6, "I3C3_SDA"),
 		STM32_FUNCTION(8, "TIM16_CH1"),
@@ -1650,7 +1650,7 @@ static const struct stm32_desc_pin stm32mp257_pins[] = {
 		PINCTRL_PIN(120, "PH8"),
 		STM32_FUNCTION(0, "GPIOH8"),
 		STM32_FUNCTION(3, "SPI1_MISO I2S1_SDI"),
-		STM32_FUNCTION(4, "SPDIFRX1_IN4"),
+		STM32_FUNCTION(4, "SPDIFRX1_IN3"),
 		STM32_FUNCTION(5, "UART4_RX"),
 		STM32_FUNCTION(7, "UART7_CTS"),
 		STM32_FUNCTION(10, "TIM5_CH1"),
@@ -1840,7 +1840,7 @@ static const struct stm32_desc_pin stm32mp257_pins[] = {
 		STM32_FUNCTION(0, "GPIOI10"),
 		STM32_FUNCTION(2, "SAI1_SCK_A"),
 		STM32_FUNCTION(3, "SPI1_SCK I2S1_CK"),
-		STM32_FUNCTION(4, "SPDIFRX1_IN1"),
+		STM32_FUNCTION(4, "SPDIFRX1_IN0"),
 		STM32_FUNCTION(5, "FDCAN2_RX"),
 		STM32_FUNCTION(6, "MDF1_CCK0"),
 		STM32_FUNCTION(9, "TIM4_CH1"),
@@ -2113,7 +2113,7 @@ static const struct stm32_desc_pin stm32mp257_pins[] = {
 		PINCTRL_PIN(160, "PK0"),
 		STM32_FUNCTION(0, "GPIOK0"),
 		STM32_FUNCTION(3, "SPI2_MISO I2S2_SDI"),
-		STM32_FUNCTION(4, "SPDIFRX1_IN3"),
+		STM32_FUNCTION(4, "SPDIFRX1_IN2"),
 		STM32_FUNCTION(6, "MDF1_CCK0"),
 		STM32_FUNCTION(8, "TIM20_ETR"),
 		STM32_FUNCTION(9, "TIM1_ETR"),
-- 
2.39.2

